lib/bitstyles_phoenix/component/form.ex

```diff
@@ -547,10 +625,44 @@ defmodule BitstylesPhoenix.Component.Form do
     <%= PhxForm.label @form, @field, @label_opts do %>
       <%= render_slot(@inner_block) %>
       <%= @label %>
+      <.required_label {assigns_to_attributes(assigns)} />
     <% end %>
     <.ui_errors form={@form} field={@field} />
     """
   end
 
   defp default_label(field), do: PhxForm.humanize(field)
+
+  defp required_label(%{required: value} = assigns)
+       when not is_nil(value) do
+    {module, function, opts} =
+      Application.get_env(:bitstyles_phoenix, :required_label, {
+        __MODULE__,
+        :default_required_label,
+        [foo: :bar]
```

Comment:
How about we make the default options an empty list?

Reply:
True, I set this for testing and then forgot 🙈

Reply:
```suggestion
        []
```